CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
   * Updated to Gradle 7, Android Gradle Plugin 7 and Rust Android Plugin 0.9 as well as building with Java 11 ([#1801](https://github.com/mozilla/glean/pull/1801))
 * iOS
   * Add support for the URL metric type ([#1791](https://github.com/mozilla/glean/pull/1791))
+  * Remove reliance on `Operation` for uploading an instead use the background capabilities of `URLSession` ([#1783](https://github.com/mozilla/glean/pull/1783))
```

Comment:
```suggestion
  * Remove reliance on `Operation` for uploading and instead use the background capabilities of `URLSession` ([#1783](https://github.com/mozilla/glean/pull/1783))
```

GitHub hid that again as "outdated", but the fix is still missing.

Reply:
Doh, I moved this to the unreleased changes section after rebasing and failed to add the missing 'd' 🤦